trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/empty.py

```diff
@@ -175,8 +174,9 @@ def empty(vehicle):
         
     # Calculating Empty Weight of Aircraft
     wt_landing_gear    = landing_gear(TOW)
-    wt_fuselage        = tube(S_fus, diff_p_fus,w_fus,h_fus,l_fus,Nlim,wt_zf,wt_wing,wt_propulsion, wing_c_r)
-    output_2           = systems(num_seats, ctrl_type, S_h, S_v, S_gross_w, ac_type)
+    
+    wt_fuselage        = tube(S_fus, diff_p_fus,w_fus,h_fus,l_fus,Nlim,wt_zf,wt_wing,wt_propulsion, wing_c_r) 
+    output_2           = systems(num_seats, ctrl_type, S_h, S_v, S_gross_w, ac_type)  
```

Comment:
What are these output numbers about? I see you didn't add them yourself but we should probably change them.